%YAML:1.0
image_width: 640
image_height: 480
camera_name: left_640
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 4.8050306563246062e+02, 0., 3.2795650986671677e+02, 0.,
       4.7947904092273950e+02, 2.3974048148295319e+02, 0., 0., 1. ]
distortion_model: plumb_bob
distortion_coefficients: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -3.7376179439012602e-01, 1.4950944626490301e-01,
       -3.5389969511980001e-03, -2.0541207419399999e-04, 0. ]
rectification_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 9.9962614058235899e-01, 2.8702242229189999e-03,
       -2.7190823402897999e-02, -2.6662765667449999e-03,
       9.9996806518772097e-01, 7.5338950082959998e-03,
       2.7211579037003001e-02, -7.4585801354259997e-03,
       9.9960187052079696e-01 ]
projection_matrix: !!opencv-matrix
   rows: 3
   cols: 4
   dt: d
   data: [ 4.3159082847111358e+02, 0., 3.5020778656005859e+02, 0., 0.,
       4.3159082847111358e+02, 2.3315425682067871e+02, 0., 0., 0., 1.,
       0. ]
